src/main/java/com/sap/oss/phosphor/fosstars/nvd/data/CpeMatch.java

```diff
@@ -1,98 +1,67 @@
 package com.sap.oss.phosphor.fosstars.nvd.data;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Optional;
+import javax.annotation.Nullable;
 import org.apache.commons.lang3.StringUtils;
 
 /**
  * CPE match string or range.
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonPropertyOrder({
-    "vulnerable",
     "cpe22Uri",
     "cpe23Uri",
-    "versionStartExcluding",
-    "versionStartIncluding",
     "versionEndExcluding",
-    "versionEndIncluding",
-    "cpe_name"
+    "versionEndIncluding"
+})
+// the properties below are ignored because they are not used
+// that saves a bit of memory
+// when they become necessary, then can be enabled
+@JsonIgnoreProperties({
+    "vulnerable",
+    "cpe_name",
+    "versionStartExcluding",
+    "versionStartIncluding"
```

Comment:
This should be "versionEndExcluding"

Reply:
Yes, `JsonIgnoreProperties` should contain "excluding" ones.